components/drafts/panel/panel_body.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useSelector} from 'react-redux';
+
+import {getCurrentRelativeTeamUrl} from 'mattermost-redux/selectors/entities/teams';
+
+import type {UserProfile, UserStatus} from '@mattermost/types/users';
+import type {PostDraft} from 'types/store/draft';
+
+import Markdown from 'components/markdown';
+import FilePreview from 'components/file_preview';
+import ProfilePicture from 'components/profile_picture';
+import {imageURLForUser, handleFormattedTextClick} from 'utils/utils';
+
+import './panel_body.scss';
+
+type Props = {
+    channelId: string;
+    displayName: string;
+    fileInfos: PostDraft['fileInfos'];
+    message: string;
+    status: UserStatus['status'];
+    uploadsInProgress: PostDraft['uploadsInProgress'];
+    userId: UserProfile['id'];
+    username: UserProfile['username'];
+}
+
+const OPTIONS = {
+    disableGroupHighlight: true,
+    mentionHighlight: false,
+};
+
+function Body({
```

Comment:
nit: Function name `PanelBody` , we are using `PanelHeader` so may be change here as well

Reply:
Done